### ISSUE TITLE
Unable to upload PDF file.

### ISSUE THREAD

**Issue by mboxraj** [RickStrahl/WestWindWebSurge#95]
Hi Rick and Team, we tried reaching out for a use case where we need to upload pdf file but we keep getting invalid content type error.
Please suggest if you have any solutions to circumvent.

Thanks
Rajesh

**Comment by RickStrahl** [comment]
This is a terrible report. You're not providing any information other than *does not work*. Please provide more info on request type etc.

Just to be sure checked this out myself:

![FileUploadWorking](https://user-images.githubusercontent.com/1374013/160928302-2de4519a-6639-4802-a72c-c0fbaee6446c.gif)

Error seems to suggest you're not setting the content type for the uploaded file correctly.

**Comment by mboxraj** [comment]
Rick apologies, I have another team member working on this part of task for file upload and also I need to check  with  office folks whether I can upload screenshots our application.  I tried the same with POSTMAN and I got the same error proving something in the controller side. From chrome browser it works  when tried F12 developers tool  the binaries are not visible in chrome. Just give me a day or 2 will debug with team members and get back to you.  Warm regards. Rajesh
